commands/copy.js

```diff
@@ -19,6 +19,7 @@ module.exports = function(program, next) {
     .option('--password [password]', 'The destination password')
     .option('--src-password [password]', 'The source password')
     .option('--dst-password [password]', 'The destination password')
+    .option('--auth-form-path [authFormPath]', 'Specify path to auth form (default: "formio") Ex. https://my.formio.io/{authFormPath}/user/login')
```

Comment:
To me this seems like we are introducing yet another option to get around a bug with our api paths. Also, the ticket associated with this does not necessarily have anything to do with "auth" but rather just a general issue related to the paths are incorrect when copying forms. This seems like a band-aid instead of an actual fix.

Reply:
@travist I've not only added this options, but also done a fix for the original issue. It's just for some probable use cases when environments may have different path to login form (ex. I found out portal.form.io uses another path for login form submission).